spring/src/main/java/org/axonframework/spring/config/AbstractAnnotationHandlerBeanPostProcessor.java

```diff
@@ -103,6 +103,19 @@ public Object postProcessAfterInitialization(final Object bean, final String bea
         return bean;
     }
 
+    /**
+     * Check if bean is NullBean
+     *
+     * {@link <a href="https://github.com/AxonFramework/AxonFramework/pull/1354">github issue</a>}
+     * @see org.springframework.beans.factory.support.NullBean#equals
+     *
+     * @param bean bean to check
+     * @return true if bean is NullBean
```

Comment:
```suggestion
     * @param bean the {@link Object} to verify if it is a {@link org.springframework.beans.factory.support.NullBean}
     * @return {@code true} if the given {@code bean} is of type {@link org.springframework.beans.factory.support.NullBean} and {@code false} otherwise
```